security-analysis/security-analysis-api/src/main/java/com/powsybl/security/limitreduction/SimpleLimitsComputer.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.security.limitreduction;
+
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.limitmodification.LimitsComputer;
+import com.powsybl.iidm.network.limitmodification.result.LimitsContainer;
+import com.powsybl.iidm.network.util.LimitViolationUtils;
+import com.powsybl.security.limitreduction.computation.SimpleLimitsReducer;
+
+import java.util.Optional;
+
+/**
+ *
```

Comment:
Could you add an explanation:
- takes a single limit reduction values for all the identifiers and applicable in pre and post contingencies;
- doesn't store the computed values in cache.